.circleci/config.yml

```
@@ -205,25 +214,34 @@ workflows:
             branches:
               only: master
           stage: development
-          assetPrefix: https://books-static.ridi.io
+          assetPrefix: https://books.ridi.io
       - deploy-s3:
           name: deploy-dev/s3
           requires:
             - build-dev
           aws-access-key-id: DEV_AWS_ACCESS_KEY_ID
           aws-secret-access-key: DEV_AWS_SECRET_ACCESS_KEY
-          s3-bucket: DEV_S3_BUCKET
+          s3-bucket: ridi-books-dev
+      - deploy:
+          name: deploy-stage
+          requires:
+            - build-dev
+          aws-access-key-id: DEV_AWS_ACCESS_KEY_ID
+          aws-secret-access-key: DEV_AWS_SECRET_ACCESS_KEY
+          stage: staging
+          assetPrefix: https://books.ridi.io
       - deploy:
           name: deploy-dev
           requires:
             - build-dev
           aws-access-key-id: DEV_AWS_ACCESS_KEY_ID
           aws-secret-access-key: DEV_AWS_SECRET_ACCESS_KEY
           stage: development
-          assetPrefix: https://books-static.ridi.io
+          assetPrefix: https://books.ridi.io
```

assetPrefix는 cloudfront 생성 이후에 변경하겠습니다.
```suggestion
          assetPrefix: https://books-static.ridi.io
```